packages/fields/package.json

```diff
@@ -66,9 +66,9 @@
     "react-dom": "^16.13.1",
     "react-select": "^3.1.0",
     "react-toast-notifications": "^2.3.0",
-    "slate": "^0.47.4",
+    "slate": "^0.58.1",
```

Comment:
FYI I've specifically left this out of my commits for now, just to hilight the problem merging in a PR that updates slate without allowing different versions between packages

Reply:
Does this supersede #2510 ?